configure.py

```diff
@@ -1132,6 +1142,16 @@ def configure_napi(output):
   version = getnapibuildversion.get_napi_version()
   output['variables']['napi_build_version'] = version
 
+  if options.arch_triplet:
+    output['variables']['arch_triplet'] = options.arch_triplet
+  else:
+    output['variables']['arch_triplet'] = 'unknown-unknown-unknown'
+
+  if options.node_relative_path:
+    output['variables']['node_relative_path'] = options.node_relative_path
+  else:
+    output['variables']['node_relative_path'] = ''
```

Comment:
Variables ending in `_path` (and `_dir` and `_file`) are treated specially by gyp and can't be empty. It should work better if you call it e.g. `node_relative_location`.

Alternatively, we could fix gyp to allow empty paths:
```diff
diff --git a/tools/gyp/pylib/gyp/input.py b/tools/gyp/pylib/gyp/input.py
index 5d4a03020a..3c992b11d0 100644
--- a/tools/gyp/pylib/gyp/input.py
+++ b/tools/gyp/pylib/gyp/input.py
@@ -2024,7 +2024,7 @@ def MakePathRelative(to_file, fro_file, item):
   #   "/' Used when a value is quoted.  If these are present, then we
   #       check the second character instead.
   #
-  if to_file == fro_file or exception_re.match(item):
+  if item == '' or to_file == fro_file or exception_re.match(item):
     return item
   else:
     # TODO(dglazkov) The backslash/forward-slash replacement at the end is a
```

Reply:
We use since two years, so it will good to keep the debian name. But you could change if needed